plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -217,6 +217,23 @@ static void log_muxer_params(struct ffmpeg_muxer *stream, const char *settings)
 	av_dict_free(&dict);
 }
 
+static void add_stream_key(struct dstr *cmd, struct ffmpeg_muxer *stream)
+{
+	obs_data_t *service;
+	const char *path_str;
+	const char *stream_key;
+
+	service = obs_output_get_service(stream->output);
```

Comment:
Create a function for getting the stream key so we don't repeat the same code.

Reply:
Done.